Documentation/InputSimulation/InputSimulationService.md

```diff
@@ -76,51 +77,57 @@ Camera position and rotation angles can be set explicitly in the tools window, a
+In the [input simulation tools window](#input-simulation-tools-window) the **Default Controller Simulation Mode** setting switches between three distinct input models. This default mode can also be set in the input simulation profile.
 
 * *Articulated Hands*: Simulates a fully articulated hand device with joint position data.
 
    Emulates HoloLens 2 interaction model.
 
    Interactions that are based on the precise positioning of the hand or use touching can be simulated in this mode.
 
-* *Gestures*: Simulates a simplified hand model with air tap and basic gestures.
+* *Hand Gestures*: Simulates a simplified hand model with air tap and basic gestures.
 
    Emulates [HoloLens interaction model](https://docs.microsoft.com/windows/mixed-reality/gestures).
 
    Focus is controlled using the Gaze pointer. The *Air Tap* gesture is used to interact with buttons.
 
-### Controlling hand movement
+* *Motion Controller*: Simulates a motion controller used with VR headsets that works similarly to far interactions with Articulated Hands.
 
-Press and hold the **Left/Right Hand Control Key** (default: *Left Shift* for left hand and *Space* for right hand) to gain control of either hand. While the manipulation key is pressed, the hand will appear in the viewport. Once the manipulation key is released, the hands will disappear after a short **Hand Hide Timeout**.
+   Emulates VR headset with controllers interaction model.
 
-Hands can be toggled on and frozen relative to the camera in the [input simulation tools window](#input-simulation-tools-window) or by pressing the **Toggle Left/Right Hand Key** (default: *T* for left and *Y* for right). Press the toggle key again to hide the hands again. To manipulate the hands, the **Left/Right Hand Control Key** needs to be held. Double tapping the **Left/Right Hand Control Key** can also toggle the hands on/off.
+   The trigger, grab and menu keys are simulated via keyboard and mouse input.
 
-Mouse movement will move the hand in the view plane. Hands can be moved further or closer to the camera using the **mouse wheel**.
+### Controlling controller movement
 
-To rotate hands using the mouse, hold both the **Left/Right Hand Control Key** (*Left Shift* or *Space*) *and* the **Hand Rotate Button** (default: *ctrl* button) and then move the mouse to rotate the hand. Hand rotation speed can be configured by changing the **Mouse Hand Rotation Speed** setting in the input simulation profile.
+Press and hold the **Left/Right Controller Control Key** (default: *Left Shift* for left controller and *Space* for right controller) to gain control of either controller. While the manipulation key is pressed, the controller will appear in the viewport. Once the manipulation key is released, the controllers will disappear after a short **Controller Hide Timeout**.
```

Comment:
As a general comment it gets kind of confusing to parse the docs when controller is repeated so often. For this part in particular, maybe something like Left/Right Controller activation key might be better?

Reply:
That was a mistake. Thanks for the catch!